org.upscayl.app.yml

```diff
@@ -0,0 +1,63 @@
+id: org.upscayl.app
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+base: org.electronjs.Electron2.BaseApp
+base-version: "22.08"
+command: upscayl-run
+separate-locales: false
+
+finish-args:
+  - --device=dri
+  - --filesystem=home
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+
+modules:
+  - name: zypak
+    sources:
+      - type: git
+        url: https://github.com/refi64/zypak
+        tag: v2022.04
+  - name: upscayl
+    buildsystem: simple
+    sources:
+      - type: file
+        only-arches:
+          - x86_64
+        url: >-
+          https://github.com/upscayl/upscayl/releases/download/v2.0.1/upscayl-2.0.1-linux.zip
+        sha256: 96883ae1bb9a3a5c0972ab7e402c5132e40c4fe0ffaeec8753524df16b03de51
+      - type: file
+        path: org.upscayl.app
+      - type: file
+        path: icon.png
+      - type: file
+        path: org.upscayl.app
+      - type: script
+        dest-filename: upscayl-run
+        commands:
+          - zypak-wrapper /app/upscayl/upscayl
+    build-commands:
+      - install -d /app/upscayl
+      - unzip upscayl-2.0.1-linux -d /app/upscayl
+      - install upscayl-run /app/bin/
+      - >-
+        install -Dm644 icon.png
+        /app/share/icons/hicolor/512x512/apps/org.upscayl.app.png
```

Comment:
"icon.png (PNG Image, 4096 × 4096 pixels) — Scaled (19%)"

Please make this a 512x512 png file. You can't put a 4096x4096 in a 512x512.

Reply:
Done

Reply:
ideally it should be from the tarball like there is a .desktop...

Reply:
Yeah, we're planning to put it in the release. I'm new to this so if we put it in the release do we have to change the file definitions?

Reply:
> Yeah, we're planning to put it in the release. I'm new to this so if we put it in the release do we have to change the file definitions?

I think we would have to change the url link and the sha256 if we updated the zip